tests/functional/cli/object/test_object.py

```diff
@@ -275,13 +275,23 @@ def test_autocontainer_object_listing(self):
         # Unpaged listing
         opts = self.get_opts([], 'json')
         output = self.openio('object list --auto --no-paging --prefix ' +
-                             prefix + ' ' + opts)
+                             prefix + ' ' + opts + ' ' + args)
         listing = self.json_loads(output)
         self.assertEqual(obj_count, len(listing))
         for obj in listing:
             # 4 columns
             self.assertEqual(4, len(obj))
 
+    def test_autocontainer_object_listing(self):
+        self._test_autocontainer_object_listing()
+
+    def test_autocontainer_object_listing_other_flatns(self):
+        self._test_autocontainer_object_listing('--flat-bits 8')
+        opts = self.get_opts([], 'json')
+        output = self.openio('container list ' + opts)
+        for entry in self.json_loads(output):
+            self.assertEqual(len(entry['Name']), 2)
```

Comment:
We should use another account for each test. Otherwise the containers are still present and this test fails.